src/App.js

```diff
@@ -70,7 +77,11 @@ class App extends React.Component {
           content = coordString;
         }
 
-        cols.push(<div className="coll" key={x}>{content}</div>);
+        let col_classes = 'coll';
+        if (dungeon) {
+          col_classes += ' ' + dungeon.colorAt(x, y);
```

Comment:
People like using https://www.npmjs.com/package/classnames for things like this.